src/viking.js

```diff
@@ -1,11 +1,96 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  receiveDamage(theDamage) {
+    this.theDamage = theDamage;
+    this.health = this.health - theDamage;
+    if (this.health > 0) {
+      return `${this.name} has received ${this.theDamage} points of damage`;
+    } else {
+      return `${this.name} has died in act of combat`;
+    }
+  }
+  battleCry() {
+    return "Odin Owns You All!";
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  constructor(health, strength) {
+    super(health, strength);
+  }
+  receiveDamage(theDamage) {
+    this.theDamage = theDamage;
```

Comment:
same here, you don't need to add theDamage as a property of Saxon, you can use it directly as a parameter:
`
`A Saxon has received ${theDamage} points of damage`;
`